### PR TITLE
Memory leak fix

### PR DESCRIPTION
**Pull request by leaanthony**
- Fixes a number of memory leaks. Significant reduction in memory usage.
  - Refactored project layout to align with latest Wails.

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -659,13 +679,17 @@ func (app *app) checkForUpdates(passive bool) {
...
-			app.runtime.Dialog.Message(&dialog.MessageDialog{
+			_, err := app.runtime.Dialog.Message(&dialog.MessageDialog{
 				Type:         dialog.InfoDialog,
 				Title:        "You're up to date",
 				Message:      fmt.Sprintf("%s is the latest version.", latest.TagName),
 				Buttons:      []string{"OK"},
 				CancelButton: "OK",
 			})
+			if err != nil {
+				// @matryer - how do you want to handle these errors?
```
Comment (matryer) [inline app/app.go:690]: not much we can do if these error is there?

Comment (leaanthony) [reply app/app.go:690]: 100% safe to ignore in this context. The error is mainly for text that cannot be converted to UTF-16.

**Comment by Etheryte** [comment]
Thank you very much for taking the time to iron this issue out. The effort is greatly appreciated. 👍

**Comment by leaanthony** [comment]
No worries! I managed to get it down to 32bytes of leakage and that was coming from foundation internals.

**Pull request merged**